programs/clearing_house/src/math/amm.rs

```diff
@@ -452,12 +463,26 @@ pub fn calculate_spreads(amm: &mut AMM) -> ClearingHouseResult<(u128, u128)> {
 
     if amm.curve_update_intensity > 0 {
         // oracle retreat
+
         // if mark - oracle < 0 (mark below oracle) and user going long then increase spread
         // msg!("amm.last_oracle_mark_spread_pct: {:?}", amm.last_oracle_mark_spread_pct);
+
         if amm.last_oracle_mark_spread_pct < 0 {
-            long_spread = max(long_spread, amm.last_oracle_mark_spread_pct.unsigned_abs());
+            long_spread = max(
+                long_spread,
+                amm.last_oracle_mark_spread_pct
```

Comment:
kinda surprising this change pops up on this pr, is this intentional?

Reply:
oh yea i actually merged this incidentally